rllib/evaluation/postprocessing.py

```diff
@@ -93,6 +93,8 @@ def compute_advantages(
     lambda_: float = 1.0,
     use_gae: bool = True,
     use_critic: bool = True,
+    rewards: TensorType = None,
+    vf_preds: TensorType = None,
```

Comment:
The changes to `compute_advantages` only make it so that we can pass rewards and vf_preds to override what's in the batch.